src/api/structure/testing-unit.ts

```diff
@@ -103,6 +107,25 @@ export default abstract class TestingUnit extends BaseUnit {
         return this.apiOrigin;
     }
 
+    private _skipJsErrors$ (options: boolean | SkipJsErrorsOptions | ((opts: SkipJsErrorsOptions) => boolean) = true, dependencies: { [key: string]: any } = {}): Function {
+        assertType([ is.boolean, is.nonNullObject, is.function ], 'skipJsErrors', 'The skipJsErrors options argument', options);
+        assertType(is.nonNullObject, 'skipJsErrors', 'The skipJsErrors dependencies argument', dependencies);
+
+        if (typeof options === 'function')
```

Comment:
You perform this logic in three places: `testcontroller`, `testingUnit` and `Runner`. Try to extract it to the separate method and use it anythere.